File: kurs programowania/lista3/cpp/Kwadrat.cpp
#include "Kwadrat.hpp"

#include <iostream>

Kwadrat::Kwadrat(int a) : Czworokat(a, a, a, a, 90) {}

double Kwadrat::pole()
{
    return a * a;
}

double Kwadrat::obwod()
{
    return 4 * a;
}

File: kurs programowania/lista3/zadanie1/Kwadrat.java
public class Kwadrat extends Czworokat {

    public Kwadrat(int a) {
        super(a, a, a, a, 90);
    }

    public double pole() {

        return this.bok1 * this.bok1;
    }

    public double obwod() {

        return 4 * this.bok1;
    }
}
